packages/nx-sonarqube/src/executors/scan/utils/utils.ts

```diff
@@ -107,16 +127,23 @@ export async function determinePaths(
             );
           }
         } else {
-          logger.warn(
-            `Skipping ${context.projectName} as it does not have a jestConfig`
-          );
+          if (executor === 'vite') {
```

Comment:
`context.projectName` should be `dep.name`

Context returns the parent app not the library name.

Reply:
Also for all warnings not just this one.

Reply:
Since it's not a change coming from this PR, I suggest opening another — dedicated — PR to tackle this.